srcts/src/imageutils/createBrush.ts

```diff
@@ -342,25 +341,29 @@ function createBrush(
     // (#1634).
     state.boundsData = mapValues(state.boundsData, (val) =>
       roundSignif(val, 14)
-    ) as BoundsData;
+    );
 
     // We also need to attach the data bounds and panel as data attributes, so
     // that if the image is re-sent, we can grab the data bounds to create a new
     // brush. This should be fast because it doesn't actually modify the DOM.
-    $div.data("bounds-data", state.boundsData);
-    $div.data("panel", state.panel);
+    $div?.data("bounds-data", state.boundsData);
+    $div?.data("panel", state.panel);
```

Comment:
```suggestion
    $div!.data("bounds-data", state.boundsData);
    $div!.data("panel", state.panel);
```
Better to fail here than silently continue